### ISSUE TITLE
Diagonal pieces for the Stand-up Coaster have bad track connections when banking or unbanking

### ISSUE THREAD

**Issue by brisingr25** [OpenRCT2/OpenRCT2#23064]
### Operating System

Windows 10, 64 bit

### OpenRCT2 build

OpenRCT2, v0.4.15 (2bcf24f3d3 on develop) provided by GitHub

### Base game

RollerCoaster Tycoon 2

### Area(s) with this issue?

This bug is a graphical glitch or error

### Describe the issue

The diagonal pieces for the stand-up coaster have bad track connections when banking or unbanking, with some gaps between the rails or offset differences. I think this is present in vanilla RCT2, though I can't check it. I also can't check the Stand-up Coaster from RCT1, but I believe it had different banking and should be alright, as should be the OpenRCT2 port of it, the Classic Stand-up Coaster.

### Steps to reproduce

The issues I've been able to find are circled in red.

![Image](https://github.com/user-attachments/assets/7f19415d-b12c-4c6a-824d-772922749051)

**Comment by Gymnasiast** [comment]
Thanks for creating the report. Could you make a TD6 of above design so we test this more easily?

**Comment by brisingr25** [comment]
Sure. Here's the design, I can't attach a TD6 so I've made it into a zip, I hope it's alright.

[Stand-up Diagonal Extravaganza.zip](https://github.com/user-attachments/files/17541613/Stand-up.Diagonal.Extravaganza.zip)

Also, while making the design I've noticed that supports for the diagonal brakes are too high and can even appear when building at ground level. If this should be in another bug issue let me know. The other coasters don't have this issue for the diagonal brakes, only this one and the Classic Stand-up as they are the same.

![Image](https://github.com/user-attachments/assets/3ad48b2e-8baa-4a73-b8f4-510681a1a7a3)
